src/com/boxboat/jenkins/library/git/GitRepo.groovy

```diff
@@ -36,22 +35,23 @@ class GitRepo implements Serializable {
 
     String getBranch() {
         if (!_branch) {
-            _branch = checkoutData?.GIT_BRANCH ?: Config.pipeline.sh(returnStdout: true, script: """
-                git rev-parse --abbrev-ref HEAD
-            """)?.trim()
-            if (_branch?.startsWith("origin/")) {
-                _branch = _branch.substring("origin/".length())
-            }
+            _branch = Config.pipeline.sh(returnStdout: true, script: "git rev-parse --abbrev-ref HEAD")?.trim()
```

Comment:
since you moved the `origin/` stripping logic to `setBranch`, go ahead and call `setBranch(Config.pipeline.sh(returnStdout: true, script: "git rev-parse --abbrev-ref HEAD")?.trim())` here so that logic gets called on this command too